crates/agglayer-storage/src/columns/epochs/transaction_hash_per_certificate_index/mod.rs

```diff
@@ -9,6 +9,7 @@ use crate::columns::PER_EPOCH_TRANSACTION_HASH_PER_CERTIFICATE_INDEX;
 /// | key                | value    |
 /// | --                 | --       |
 /// | `CertificateIndex` | `Hash`   |
+#[allow(unused)]
 pub struct TransactionHashPerCertificateIndexColumn;
```

Comment:
Hmm is this db column actually used? I don't see any `db.(get|put|keys)<TransactionHashPerCertificateIndexColumn>`. For transaction hash where a certificate was settled, there is a filed in `CertificateHeader` IIRC.

Reply:
Filed a PR to clean this up:
* #845